content/blog/2024/01/25/2024-01-25-jenkins-2023-recap.adoc

```diff
@@ -0,0 +1,231 @@
+---
+layout: post
+title: "Jenkins 2023 Recap"
+tags:
+- jenkins
+- newsletter
+- community
+- contribute
+authors:
+- dduportal
+- markewaite
+- gounthar
+- wadeck
+- kmartens27
+- alyssat
+opengraph:
+  image: /images/post-images/2024/01/25/image1.png
+links:
+discourse: true
+---
+
+image:/images/post-images/2024/01/25/image1.png[Jenkins 2023 recap]
+
+link:/blog/2023/01/12/jenkins-newsletter/[Here is 2022 Recap.]
```

Comment:
```suggestion
link:/blog/2023/01/12/jenkins-newsletter/[Here is the 2022 Recap.]
```
Is this supposed to be here? I considered it a reference to use for composing this years recap, as opposed to something that should be included. However, I could be totally off base on this.

@alyssat should the 2022 recap be included here, or should we just focus on 2023?

Reply:
Ooopsie.
I hesitated, as it looked odd to me. 🤷